alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/DefaultConnectionInfoRepository.java

```diff
@@ -78,6 +78,7 @@ public ListenableFuture<ConnectionInfo> getConnectionInfo(
 
               return new ConnectionInfo(
                   info.getIpAddress(),
+                  info.getPublicIpAddress(),
```

Comment:
What if an instance does not have a Public IP, does this fail nicely...?

Reply:
Now it raises an RuntimeException when IP address is not provided. Thanks for you review.